vnext/include/ReactUWP/IReactInstance.h

```diff
@@ -29,6 +29,8 @@ struct ReactInstanceSettings
   bool UseLiveReload { false };
   bool UseDirectDebugger{ false };
   bool UseJsi { true };
+  bool EnableJITCompilation { true };
```

Comment:
I get better startup times with this set to false. but all other places default to true. So here I put true for consistency.
If consensus is that false is always better for chakraRT I can change to false everywhere